src/components/CardAddress/index.tsx

```diff
@@ -12,45 +14,24 @@ interface CardAddressProps extends HTMLAttributes<HTMLInputElement> {
   complement?: string
 }
 
-const RadioButtonLabel = styled.label`
+const Flex = styled.div`
   display: flex;
   align-items: center;
+  gap: 16px;
```

Comment:
get size by `Spacing` token

Reply:
This space cannot be the `Spacing` token, it is in rem and needs to be pix